Errors building foxglove-bridge 0.7.7-1 with Yocto (kirkstone) and ros2 humble

**Description**
When building with yocto the following error is seen

`
/home/axcend/OSTL-k/build-openstlinuxweston-stm32mp1/tmp-glibc/work/cortexa7t2hf-neon-vfpv4-oe-linux-gnueabi/foxglove-bridge/0.7.7-1-r0/git/ros2_foxglove_bridge/src/ros2_foxglove_bridge.cpp:482:50: error: no matching function for call to 'max(long unsigned int, size_t)'
  482 |     const size_t publisherHistoryDepth = std::max(1ul, qos.depth());
`

- Version: 0.7.7-1
- Platform: target is arm-oe-linux-gnueabi

**Steps To Reproduce**
<!--- Include the minimum possible code or example data to reproduce the problem. -->
bitbake foxglove-bridge

**Expected Behavior**
no build error

--- a/ros2_foxglove_bridge/src/ros2_foxglove_bridge.cpp
+++ b/ros2_foxglove_bridge/src/ros2_foxglove_bridge.cpp
@@ -480,7 +480,7 @@ void FoxgloveBridge::subscribe(foxglove::ChannelId channelId, ConnectionHandle c
     // broadcasters). See also
     // https://github.com/foxglove/ros-foxglove-bridge/issues/238 and
     // https://github.com/foxglove/ros-foxglove-bridge/issues/208
-    const size_t publisherHistoryDepth = std::max(1ul, qos.depth());
+    const size_t publisherHistoryDepth = std::max(static_cast<size_t>(1), qos.depth());
     depth = depth + publisherHistoryDepth;
   }
 

